client/js/utils.js

```diff
@@ -83,17 +82,8 @@ function join(args) {
 	}
 }
 
-function toggleNickEditor(toggle) {
-	$("#nick").toggleClass("editable", toggle);
-	$("#nick-value").prop("contenteditable", toggle);
-}
-
 function setNick(nick) {
-	// Closes the nick editor when canceling, changing channel, or when a nick
-	// is set in a different tab / browser / device.
-	toggleNickEditor(false);
-
-	$("#nick-value").text(nick);
+	$("#nick").text(nick);
```

Comment:
I believe this 1-line function is now called in 2 places left, and I don't think we'll have any further case. Move its content to where it's called instead?